apps/src/componentLibrary/breadcrumbs/_Breadcrumb.tsx

```diff
@@ -0,0 +1,81 @@
+import classNames from 'classnames';
+import React, {useEffect, useRef, memo} from 'react';
+
+import moduleStyles from './breadcrumbs.module.scss';
+
+const commonI18n = require('@cdo/locale');
```

Comment:
Minor: you should be able to do `import commonI18n from '@cdo/apps/types/locale` to get the same object but fully typed.